core/templates/dev/head/pages/topic_editor/TopicEditor.ts

```diff
@@ -76,15 +76,28 @@ require('pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.ts');
 
 require('pages/topic_editor/TopicEditorStateService.ts');
 require('services/contextual/UrlService.ts');
+require('services/PageTitleService.ts');
 
 oppia.constant('INTERACTION_SPECS', GLOBALS.INTERACTION_SPECS);
 
 oppia.constant(
   'TOPIC_NAME_INPUT_FOCUS_LABEL', 'topicNameInputFocusLabel');
 
+oppia.constant('EVENT_TOPIC_INITIALIZED', 'topicInitialized');
+oppia.constant('EVENT_TOPIC_REINITIALIZED', 'topicReinitialized');
```

Comment:
Don't add these constants again, they are defined in `TopicEditorStateService.ts`.

Reply:
done